wordpress-mobile/WordPress-Android
# Jetpack promo screenshots strings configuration

Parent #14633

This PR adds the logic for generating/downloading the string text files for creating Jetpack promo screenshots.
Changes include:
-  Added screenshot text files to WordPress/jetpack_metadata
-  Fastfile lane: `update_jetpack_appstore_strings` - Added screenshot text files
- ScreenshotFastfile lane: `download_promo_strings` - Add logic to download the translated strings for Jetpack
- Added translated string files to project `fastlane/playstoreres/jetpack_metadata`

I started separating out WP vs. JP parameters and moving them to the top of `ScreenshotFastfile`. WPAndroid parameters are prefixed with `WP` and Jetpack with `JP`.

**NOTE**: This work was extracted from draft PR #14744 because that PR was becoming too hard to follow. The remainder of that draft PR will also be extracted into separate PRs.

**Merge Instructions**
~~1. Ensure that PR #15062 has been merged, as this PR builds off of that one.~~
~~2. Ready Not Ready For Merge Label~~
3. Merge as normal

**To test:**
- Download this branch
- Delete the `fastlane/playstoreres/jetpack_metadata` folder
- Open terminal and run
```
bundle install
bundle exec fastlane download_promo_strings app:jetpack
```
- Note that the `jetpack_metadata` folder has been recreated and the locale strings have been downloaded to the proper subfolders

## Regression Notes
1. Potential unintended areas of impact
String files are not created in the correct directory

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual ran `download_promo_strings`

3. What automated tests I added (or what prevented me from doing so) 
N/A

PR submission checklist:
